modules/scheduler/scheduler/src/main/java/org/motechproject/scheduler/service/MotechSchedulerService.java

```diff
@@ -1,7 +1,6 @@
 package org.motechproject.scheduler.service;
```

Comment:
add missing description to param in javadoc, for example in method 'safeUnscheduleRunOnceJob'. Every param should have brief description.